application-licensing-licensor/application-licensing-licensor-ui/src/main/resources/Licenses/Code/LicensesNotificationsUIX.xml

```diff
@@ -38,43 +38,186 @@
   <hidden>true</hidden>
   <content>{{include reference="Licenses.Code.VelocityMacros" /}}
 
+{{velocity output="false"}}
+#macro(renderNotification $btnClass $notificationClass $translation $extensionNames $showMore)
+  &lt;$licensesNotificationWrapperTagName class="notifications-missingLicenses $notificationClass"&gt;
```

Comment:
You are still using `missingLicenses` in the class name. Can you also make this one more generic?